config.json

```diff
@@ -107,9 +107,15 @@
       "topics": []
     },
     {
-      "difficulty": 1,
+      "difficulty": 2,
```

Comment:
For me I've been adding the difficulty and topics at the same time as the process of making those judgements is intrinsically combined. I'll look at other peoples submissions and see if there are any comments indicating peoples feelings regarding the exercise, so that the difficulty rating is not confined to my own experience. This is especially important for exercises that I completed a while back where maybe I've forgotten some of the pain of not knowing the solution. At the same time I'm looking for what topics are being covered. For some exercises the submissions are all very similar, and so the topics list and difficulty is quite easy. For others, such as this one, there can be a number of different approaches, covering different topics. 

I think it would be a more difficult task to rate all the exercises in one fell swoop and then add topics as we go. Would someone need to have completed all the exercises to do this? If they're reviewing the exercise to gauge difficulty, is it not worth making a note of topics simultaneously? Or would the initial difficulty ratings be a quick and simple best guess effort, to be adjusted over time? (My concern with this is starting an exercise's rating with a misleading level of difficulty) My intention from here is to add topics to exercises in the current order they are in, I only added grains out of step as I completed it yesterday and wanted to tackle it while fresh in my mind.

I agree with @petertseng, it's hard to make a judgement on order until there are adjacent exercises to compare against.

Also to consider is the idea of this exercise being simple in one way, but potentially introducing a difficult concept. 

I've updated the config.json in the xpython track in a similar way to this, and my assumption was that any reordering could occur when there's sufficient information to compare the exercises. I wanted to get started on this to at least have a kind of baseline from which to work from. Anyway I've rambled on, sorry.